### PR TITLE
[MIG] l10n_ar_purchase: Migration to 18.0

### PR DESCRIPTION
**Pull request by pablohmontenegro**

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/odoo-argentina/pull/938).

File: l10n_ar_purchase/views/purchase_report_templates.xml [hunk 1]
```diff
@@ -61,7 +61,7 @@
         <t t-set="address" position="replace"/>
 
         <!-- remove default document title -->
-        <h2 position="replace">
+        <xpath expr="//t[@t-set='layout_document_title']" position="replace">
```
Comment (pablohmontenegro) [inline l10n_ar_purchase/views/purchase_report_templates.xml:64]:
```suggestion
        <t t-set='layout_document_title']" position="replace">
```

Comment (pablohmontenegro) [inline l10n_ar_purchase/views/purchase_report_templates.xml]:
```suggestion
        <t t-set='layout_document_title' position="replace">
```

Comment (mem-adhoc) [inline l10n_ar_purchase/views/purchase_report_templates.xml]:
```suggestion
        <t t-set='layout_document_title' position="replace">
```

File: l10n_ar_purchase/views/purchase_report_templates.xml [hunk 2]
```diff
@@ -109,7 +109,7 @@
                 </div>
 
             </div>
-        </h2>
+        </xpath>
```
Comment (mem-adhoc) [inline l10n_ar_purchase/views/purchase_report_templates.xml]:
```suggestion
        </t>
```

Comment (pablohmontenegro) [inline l10n_ar_purchase/views/purchase_report_templates.xml]:
```suggestion
        </t>
```

File: l10n_ar_purchase/views/purchase_report_templates.xml [hunk 3]
```diff
@@ -146,9 +146,8 @@
         <t t-set="address" position="replace"/>
 
         <!-- remove default document title -->
-        <h2 position="replace"/>
-        <h2 position="replace"/>
-        <h2 position="replace"/>
+        <xpath expr="//div[hasclass('mt-4')]/t[@t-set='layout_document_title']" position="replace">
+        </xpath>
```
Comment (mem-adhoc) [inline l10n_ar_purchase/views/purchase_report_templates.xml]: idem

File: l10n_ar_purchase/views/purchase_report_templates.xml [hunk 4]
```diff
@@ -146,9 +146,8 @@
         <t t-set="address" position="replace"/>
 
         <!-- remove default document title -->
-        <h2 position="replace"/>
-        <h2 position="replace"/>
-        <h2 position="replace"/>
+        <xpath expr="//div[hasclass('mt-4')]/t[@t-set='layout_document_title']" position="replace">
```
Comment (pablohmontenegro) [inline l10n_ar_purchase/views/purchase_report_templates.xml]:
```suggestion
        <t t-set='layout_document_title' position="replace">
```

**Review by mem-adhoc — CHANGES_REQUESTED** [review]

**Review by zaoral — APPROVED** [review]
@roboadhoc r+

**Pull request closed without merging**